backend/src/main/java/com/dccsacademy/resources/RootRedirectResource.java

```diff
@@ -0,0 +1,23 @@
+package com.dccsacademy.resources;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.core.Response;
+
+@Path("/")
+public class RootRedirectResource {
```

Comment:
With Quinoa you do not need redirect resource. There is config to separate requests for frontend and requests for backend resource. Here is configuration reference: https://docs.quarkiverse.io/quarkus-quinoa/dev/config-reference.html